packages/babel-helpers/src/helpers.js

```diff
@@ -618,33 +618,33 @@ helpers.interopRequireDefault = helper("7.0.0-beta.0")`
   }
 `;
 
-helpers.interopRequireWildcard = helper("7.0.0-beta.0")`
-  function _getRequireWildcardCache() {
+helpers.interopRequireWildcard = helper("7.14.0")`
+  function _getRequireWildcardCache(i) {
     if (typeof WeakMap !== "function") return null;
 
-    var cache = new WeakMap();
-    _getRequireWildcardCache = function () { return cache; };
-    return cache;
+    var cache = [/* babel interop */ new WeakMap(), /* node interop */ new WeakMap()];
+    _getRequireWildcardCache = function (i) { return cache[i]; };
+    return cache[i];
   }
 
-  export default function _interopRequireWildcard(obj) {
-    if (obj && obj.__esModule) {
+  export default function _interopRequireWildcard(obj, nodeInterop) {
+    if (!nodeInterop && obj && obj.__esModule) {
       return obj;
     }
 
     if (obj === null || (typeof obj !== "object" && typeof obj !== "function")) {
       return { default: obj }
     }
 
-    var cache = _getRequireWildcardCache();
+    var cache = _getRequireWildcardCache(+!!nodeInterop);
```

Comment:
Nit: Are we trying to save bytes? It'd be easier to grok with a key name, instead of boolean->int conversion.

Reply:
Yes but given we usually don't worry too much about that I can make it more readable.